### ISSUE TITLE
imp dependency?  [No module named 'imp']

### ISSUE THREAD

**Issue by RLinnae** [pescadores/pescador#163]
Hi, trying to install pescador for python 3.7 3.10.11 or 3.11.5 and receive the error

````
Collecting pescador
  Using cached pescador-2.1.0.tar.gz (20 kB)
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [6 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "C:\Users\rober\AppData\Local\Temp\pip-install-oyh88l5c\pescador_deb74d8372d141c3afc979dfba7f267a\setup.py", line 3, in <module>          import imp
      ModuleNotFoundError: No module named 'imp'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
````

I have tried upgrading pip / wheel / clearing pip cache - same issue.

**Comment by fpc987** [comment]
I have the same issue.

**Comment by fpc987** [comment]
i bypass the imp module by download  https://pypi.org/project/pescador/#files .
 remove the import of imp and hard code the version in setup.py

```
#import imp
#version = imp.load_source('pescador.version', 'pescador/version.py')

setup(
    name='pescador',
    version='2.1.0', #version.version,
```

**Comment by bmcfee** [comment]
Fixed by #164

**Comment by bmcfee** [comment]
Just a heads up that we finally pushed out 3.0, so you should be good to update this now.
